test/src/main/scala/scalarules/test/scala_import/BUILD

```diff
@@ -5,8 +5,8 @@ load("//scala:scala_import.bzl", "scala_import")
 scala_import(
     name = "guava_and_commons_lang",
     jars = [
-        "@com_google_guava_guava_21_0_with_file//jar:file",
-        "@org_apache_commons_commons_lang_3_5//jar:file",
+        "@maven//:v1/https/jcenter.bintray.com/com/google/guava/guava/21.0/guava-21.0.jar",
+        "@maven//:v1/https/jcenter.bintray.com/org/apache/commons/commons-lang3/3.5/commons-lang3-3.5.jar",
```

Comment:
do we really need to use the source of the binary in the name? What about users that use private internal resolvers?

Reply:
Filed https://github.com/bazelbuild/rules_jvm_external/issues/284

There's currently the ability to reference maven_install artifacts with the `//jar`-style labels with [`generate_compat_repositories`](https://github.com/bazelbuild/rules_jvm_external#repository-aliases), but we didn't implement the `file` label. We can definitely do that - will look into it very soon.